kubemarine/procedures/check_iaas.py

```diff
@@ -288,6 +288,30 @@ def system_distributive(cluster):
         
         tc.success(results=", ".join(detected_supported_os))
 
+def check_kernel_version(cluster):
+    """
+    This method compares the linux kernel version with the bad version
+    """
+    with TestCase(cluster.context['testsuite'], '015', "Software", "Kernel version") as tc:
+        bad_results = []
+        bad_kernel_ubuntu = ['5.4.0-132-generic']
```

Comment:
Why do not use `globals.yaml` for that purpose?

Reply:
I think to store invalid versions in global.yaml is not a good idea. Because why store invalid data in global.yaml

Reply:
`globals` have two aims - 1) to refer to it from inventory (defaults.yaml) and 2) to store constants. We do not need to refer to it from inventory. And storing of constants in python code has other benefits - their usages are highlighted by IDE.
So I tend to leave it in check_iaas.py but probably declare it in global level, not locally in the check method.

Reply:
Why the kernel versions list can't be some constant?

Reply:
bad_kernels are now in global.yaml